service/routes.py

```diff
@@ -19,7 +19,11 @@
 def index():
     """ Root URL response """
     return (
-        "Reminder: return some useful information in json format about the service here",
+        jsonify(
+            name="Orders system REST API Service",
+            version="1.0",
```

Comment:
Please update the test to verify the name and version returned by the service.

Reply:
Is this really necessary? We may change the name and version a lot

Reply:
I see, yeah it was just an improvement. LGTM!